Formula/p/pistache.rb

```diff
@@ -0,0 +1,158 @@
+class Pistache < Formula
+  desc "Modern, fast, elegant HTTP + REST C++17 framework with pleasant API"
+  homepage "https://github.com/pistacheio/pistache"
+  url "https://github.com/pistacheio/pistache/archive/refs/tags/v0.4.26.tar.gz"
+  sha256 "29af6562547497acf6f49170661786fe8cf1ed3712ad80e69c53da4661c59544"
+  license "Apache-2.0"
+  head "https://github.com/pistacheio/pistache.git", branch: "master"
+
+  depends_on "cmake" => :build # for howard-hinnant-date
+  depends_on "cpp-httplib" => :build
+  depends_on "googletest" => :build
+  depends_on "howard-hinnant-date" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
+  depends_on "rapidjson" => :build
+  depends_on "brotli"
+  depends_on "libevent"
+  depends_on "openssl@3"
+  depends_on "zstd"
+
+  uses_from_macos "curl" => :build
+  uses_from_macos "zlib"
+
+  def install
+    system "meson", "setup", "build",
+           "-DPISTACHE_USE_SSL=true", "-DPISTACHE_BUILD_EXAMPLES=true",
+           "-DPISTACHE_BUILD_TESTS=true",
+           "-DPISTACHE_BUILD_DOCS=false",
+           "-DPISTACHE_USE_CONTENT_ENCODING_DEFLATE=true",
+           "-DPISTACHE_USE_CONTENT_ENCODING_BROTLI=true",
+           "-DPISTACHE_USE_CONTENT_ENCODING_ZSTD=true",
+           *std_meson_args
```

Comment:
```suggestion
    system "meson", "setup", "build",
                    "-DPISTACHE_USE_SSL=true",
                    "-DPISTACHE_BUILD_EXAMPLES=false",
                    "-DPISTACHE_BUILD_TESTS=false",
                    "-DPISTACHE_BUILD_DOCS=false",
                    "-DPISTACHE_USE_CONTENT_ENCODING_DEFLATE=true",
                    "-DPISTACHE_USE_CONTENT_ENCODING_BROTLI=true",
                    "-DPISTACHE_USE_CONTENT_ENCODING_ZSTD=true",
                    *std_meson_args
```

1. Follow the style used by other formulae (see, e.g., `libgusb` or `libjcat`)
2. Skip building the tests or examples since it doesn't look like we're using/running them anywhere

Reply:
@carlocab and @chenrui333 - changed and pushed again